src/main/g8/src/main/scala/com/example/helloworld/GreeterServer.scala

```diff
@@ -37,7 +37,7 @@ object GreeterServer {
 
   def main(args: Array[String]): Unit = {
     // important to enable HTTP/2 in ActorSystem's config
-    val conf = ConfigFactory.parseString("pekko.http.server.enable-http2 = on")
+    val conf = ConfigFactory.parseString("pekko.http.server.preview.enable-http2 = on")
```

Comment:
Can we change the config so it works in new version?

Reply:
we need the quickstart to work with the current version - we can't just wait for a future release